src/sass/app_2/_common/components/calendar.scss

```diff
@@ -145,18 +144,82 @@
         }
         &--today {
             font-weight: 400;
-            border: 1px solid $COLOR_LIGHT_BLACK_1;
-            border-radius: 5px;
+            background: $COLOR_DARK_BLUE_5;
+
+            &:hover {
+                background: $COLOR_DARK_BLUE_5 !important;
+            }
+            &:before {
+                position: absolute;
+                content: '';
+                width: 100%;
+                left: 50%;
+                z-index: -1;
+                height: 100%;
+                padding-right: 10px;
+                background: $COLOR_DARK_BLUE_5 !important;
+            }
         }
         &--active {
             color: $COLOR_WHITE;
-            background: $COLOR_LIGHT_GRAY_1;
+            background: $COLOR_DARK_BLUE_5 !important;
             font-weight: bold;
 
             &:hover {
                 background: $COLOR_LIGHT_GRAY_1;
             }
         }
+        &--between {
+            background: $COLOR_DARK_BLUE_5;
```

Comment:
here too, better to use light theme color here and change them to dark color in the dark theme color class